client/my-sites/plugins/main.jsx

```diff
@@ -466,7 +466,7 @@ export class PluginsMain extends Component {
 			<>
 				<DocumentHead title={ pageTitle } />
 				<QueryPlugins siteId={ selectedSite?.ID } />
-				<QuerySiteFeatures siteIds={ this.props.siteIds } />
+				<QueryJetpackSitesFeatures />
```

Comment:
Also, let's make sure we test this thoroughly with a WP.com site. I just checked and I can access `/plugins/manage/:site` for a WP.com simple site, which doesn't make much sense to me. We used to list a pseudo-list of plugins that are always available on WP.com simple sites, but now we just list an empty list of installed plugins and redirect after a few seconds. Sounds like this might be a side effect of recent refactors and might need additional handling for simple sites.

Reply:
I created an Asana card for this: 1202619025189113-as-1203512513500111